bin/component-test-runner

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+DEBUG=1 DATABASE_URL=postgres://localhost:5432/posthog_e2e_test python manage.py setup_dev &
```

Comment:
This is not needed - component tests do not rely on there being a test database.